draft-ietf-tls-esni.md

```diff
@@ -797,9 +797,8 @@ the extension syntactically and abort the connection with a "decode_error" alert
 if it is invalid. It otherwise ignores the extension and MUST NOT use the retry
 keys.
 
-[[OPEN ISSUE: if the client sends a GREASE "encrypted_client_hello" extension,
-should it also send a GREASE "pre_shared_key" extension? If not, GREASE+ticket
-is a trivial distinguisher.]]
+[[TODO: Provide a defense for the GREASE+resumption and GREASE+PSK
```

Comment:
Let's revert this. We don't yet have consensus as to whether or not we want (or need) to mitigate this particular differentiator. (Please also file a separate issue focused specifically on this issue that we can use to track discussion!)

Reply:
(#354 seems too generic to be useful for converging on consensus here, I think.)

Reply:
Done. I also added a reference to #384.